lib/db/Migration.ts

```diff
@@ -553,11 +560,103 @@ class Migration {
         break;
       }
 
+      case 11: {
+        await this.sequelize
+          .getQueryInterface()
+          .addColumn(Swap.tableName, 'preimage', {
+            type: new DataTypes.STRING(64),
+            allowNull: true,
+            unique: true,
+          });
+
+        this.toBackFill.push(11);
+        await this.finishMigration(versionRow.version, currencies);
+        break;
+      }
+
       default:
         throw `found unexpected database version ${versionRow.version}`;
     }
   };
 
+  public backFillMigrations = async (currencies: Map<string, Currency>) => {
+    for (const version of this.toBackFill) {
+      this.logger.info(
+        `Starting backfilling of migration of database schema version ${version}`,
+      );
+
+      switch (version) {
+        case 11: {
+          await this.logProgress(
+            Swap.tableName,
+            100,
+            await LightningPaymentRepository.findByStatus(
+              LightningPaymentStatus.Success,
+            ),
+            async (payment) => {
+              const { base, quote } = splitPairId(payment.Swap.pair);
+              const lightningSymbol = getLightningCurrency(
+                base,
+                quote,
+                payment.Swap.orderSide,
+                false,
+              );
+              const currency = currencies.get(lightningSymbol);
+              if (currency === undefined) {
+                this.logger.warn(
+                  `Could not get lightning currency ${lightningSymbol} for ${swapTypeToPrettyString(payment.Swap.type)} Swap ${payment.Swap.id}`,
+                );
+                return;
+              }
+
+              let preimage: string | undefined = undefined;
+              if (
+                payment.node === NodeType.LND &&
+                currency.lndClient !== undefined
+              ) {
+                const res = await currency.lndClient.trackPayment(
+                  getHexBuffer(payment.Swap.preimageHash),
+                );
+                if (
+                  res !== undefined &&
+                  res.paymentPreimage !== undefined &&
+                  res.paymentPreimage.length > 0
```

Comment:
```suggestion
                  res?.paymentPreimage?.length > 0
```
Wouldn't that be sufficient?